packages/forma-36-react-components/src/components/Button/Button.stories.tsx

```diff
@@ -12,7 +13,7 @@ export default {
     propTypes: [Button['__docgenInfo']],
   },
   argTypes: {
-    icon: { control: { type: 'select', options: iconName } },
+    icon: { control: { disable: true } },
```

Comment:
Why?
If you disable this control, people won't be able to try different icons in the button

Reply:
Kind-of-but-not-really a stopgap. There's no easy way to import all icons and list them without having to update the list manually when new icons are added.

Reply:
Ahhhh I see.... ok it makes sense then